CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+- Enabled authenticators can now be configured via a configuration file, the
+  CONJUR_AUTHENTICATORS environment variable, or the authenticator allowlist
+  API.
+  [cyberark/conjur##2173](https://github.com/cyberark/conjur/issues/#2173)
```

Comment:
@h-artzi i'm probably missing something but i couldn't find in the code how authenticators can still be enabled by the CONJUR_AUTHENTICATORS environment variable. can you point me to it?

also - i am not sure that the authenticator allowlist  API is GA so not sure that it should be in the changelog. @alexkalish wdyt?

Reply:
The [anyway_config](https://github.com/palkan/anyway_config#environment-variables) gem takes care of the usage of the environment variable CONJUR_AUTHENTICATORS.